### DISCUSSION TITLE
@types/meteor missing dependency on @types/react, allowing error TS2694: Namespace 'React' has no exported member 'DependencyList'.

### DISCUSSION THREAD

**Discussion by KeithGillette** [DefinitelyTyped/DefinitelyTyped#54927] [Issues with a @types package] [answered]
Per @radekmie [in the Meteor forum](https://forums.meteor.com/t/meteor-2-typescript-type-definitions/56344/7?u=keithgillette), `@types/meteor` is missing a dependency on `@types/react`, which can lead to a `TS2694: Namespace 'React' has no exported member 'DependencyList'.` error in projects not already including the expected `@types/react` version.

**Comment by radekmie** [comment, marked as answer]
I filed a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55157.

**Reply by radekmie** [reply]
As said in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55157#issuecomment-901035795, there's a workaround and we'll describe it in the readme instead.
